### PR TITLE
Add canvas rendering, rename chunks/frames to reflect latest

### PR DESCRIPTION
**Pull request by chcunningham**

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -55,28 +55,28 @@ Provide web apps with efficient access to built-in (software and hardware) media
 
 We build on top of [WHATWG Streams](https://streams.spec.whatwg.org/) (in particular [TransformStreams](https://streams.spec.whatwg.org/#ts-class)) and [MediaStreamTracks](https://www.w3.org/TR/mediacapture-streams/#dom-mediastreamtrack).
```
Comment (sandersdan) [inline explainer.md]: I wouldn't say that the current design is built on MediaStreamTracks, just that we hope to integrate well with them.

File: explainer.md [hunk 2]
```diff
@@ -55,28 +55,28 @@ Provide web apps with efficient access to built-in (software and hardware) media
...
 
-**EncodedAudioPacket**s and **EncodedVideoFrame**s provide access to codec-specific encoded media bytes so they may be transported, saved, etc.
+**EncodedAudioChunk**s and **EncodedVideoChunk**s provide access to codec-specific encoded media bytes so they may be transported, saved, etc.
 
-**DecodedAudioPacket**s and **DecodedVideoFrame**s provide opaque handles for passing to/from other APIs (such as to/from MediaStreamTracks).   
+**AudioPacket**s and **VideoFrame**s may be passed opaque handles to other APIs (such as to/from MediaStreamTracks). Alternatively, AudioPacket may expose an [AudioBuffer](https://webaudio.github.io/web-audio-api/#audiobuffer) for rendering via [AudioWorklet](https://webaudio.github.io/web-audio-api/#audioworklet), while VideoFrame may expose an ImageBitmap for efficient rendering to canvas.
```
Comment (sandersdan) [inline explainer.md]: "may be passed opaque handles": passed *as* opaque handles?

It's increasingly looking like they will not be opaque, maybe just stick with the second part.

File: explainer.md [hunk 3]
```diff
@@ -55,28 +55,28 @@ Provide web apps with efficient access to built-in (software and hardware) media
...
 
-A **VideoDecoder** is a TransformStream from EncodedVideoFrame to DecodedVideoFrame.
+A **VideoDecoder** is a TransformStream from EncodedVideoChunk to VideoFrame.
 
-A **VideoTrackWriter** converts a WritableStream of DecodedVideoFrame into a MediaStreamTrack.
+A **VideoTrackWriter** converts a WritableStream of VideoFrame into a MediaStreamTrack.
 
 ## Examples
-### Example of decode for low-latency live streaming or cloud gaming 
+### Example of decode and rendering via \*TrackWriters for low-latency live streaming or cloud gaming
```
Comment (sandersdan) [inline explainer.md]: What does the asterisk mean?

Recommend moving below the canvas path.

File: explainer.md (hunk 1 above)
Comment (chcunningham) [reply explainer.md]: Done

File: explainer.md (hunk 2 above)
Comment (chcunningham) [reply explainer.md]: Done. Just kept the second part.

Comment (chcunningham) [reply explainer.md]: Actually, going back to keeping it with the typo fixed (your first suggestion). Not that I disagree about  the overall direction, but we still have TrackWriters in the explainer and to some extent landed in Chrome... so seems sane to keep it for now.

File: explainer.md (hunk 3 above)
Comment (chcunningham) [reply explainer.md]: Asterisk just a typo. Move complete.

File: explainer.md (hunk 2 above)
Comment (chcunningham) [reply explainer.md]: I re-ordered them to put canvas/audioworklet first.

**Pull request merged**